IFC4x3/Properties/s/SystemCapacity_37sz88SXT3JOvbrpAb4kbe/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SystemCapacity_37sz88SXT3JOvbrpAb4kbe" Name="SystemCapacity" UniqueId="c7dbd208-7217-434d-8e65-d732a512e968" PrimaryDataType="IfcIntegerCountRateMeasure" />
```

Comment:
firstly Shouldnt this be `EquipmentCapacity` not system. secondly isnt it just bandwidth for you guys does that mean something else